app/src/main/java/com/github/se/travelpouch/ui/notifications/NotificationsScreen.kt

```diff
@@ -0,0 +1,179 @@
+package com.github.se.travelpouch.ui.notifications
+
+import android.annotation.SuppressLint
+import android.widget.Toast
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.itemsIndexed
+import androidx.compose.material3.Button
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBar
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.github.se.travelpouch.model.activity.ActivityViewModel
+import com.github.se.travelpouch.model.documents.DocumentViewModel
+import com.github.se.travelpouch.model.events.EventViewModel
+import com.github.se.travelpouch.model.notifications.Notification
+import com.github.se.travelpouch.model.notifications.NotificationViewModel
+import com.github.se.travelpouch.model.profile.ProfileModelView
+import com.github.se.travelpouch.model.travels.ListTravelViewModel
+import com.github.se.travelpouch.ui.navigation.BottomNavigationMenu
+import com.github.se.travelpouch.ui.navigation.NavigationActions
+import com.github.se.travelpouch.ui.navigation.TopLevelDestinations
+import java.util.Calendar
+import java.util.Date
+
+@OptIn(ExperimentalMaterial3Api::class)
+@SuppressLint("UnusedMaterial3ScaffoldPaddingParameter")
+@Composable
+fun NotificationsScreen(
+    navigationActions: NavigationActions,
+    notificationViewModel: NotificationViewModel,
+    profileModelView: ProfileModelView,
+    listTravelViewModel: ListTravelViewModel,
+    activityViewModel: ActivityViewModel,
+    documentViewModel: DocumentViewModel,
+    eventsViewModel: EventViewModel
+) {
+  profileModelView.getProfile()
+
```

Comment:
wouldn't it be better to put the profileModelView.getProfile() in a LaunchedEffect  to get the profile only when it changes and not each time the screen is recomposed ?

```kotlin 

 LaunchedEffect(profile) {
        profileModelView.getProfile()
    }
    
```